addons/test_mail/tests/test_performance.py

```diff
@@ -419,6 +419,51 @@ def test_mail_composer_mass_w_template(self):
             composer._action_send_mail()
 
         self.assertEqual(len(self._new_mails), 10)
+    @users('admin', 'employee')
+
+    @warmup
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink', 'odoo.tests')
+    def test_mail_composer_mass_w_template_report(self):
```

Comment:
Probably not interesting anymore but let me know since it's already here

Reply:
Did you ever find out? ^^

Is this testing the performances of the duplication detection?
I'm guessing if it does not alter any of the other query counts, that is proof enough that it does not affect it, but maybe this code adds specific cases not covered elsewhere and specific to duplication checks?

Reply:
I believe this was when I checked some of the fields of the attachments to make a guess as to whether it should be considered a duplicate. And this was to make sure I didn't add a ridiculous number of queries for larger numbers of recipients

But since now I never access anything now it's less relevant.

It won't cover any branch that's not already covered so I guess I'll remove it.